src/Auth/AuthConfig.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+use Tempest\Auth\Authenticators\DatabaseAuthenticator;
+use Tempest\Auth\Contracts\Authenticator;
+
+final class AuthConfig
+{
+    public function __construct(
+        /** @var class-string<Authenticator> */
+        public string $authenticatorClass = DatabaseAuthenticator::class,
```

Comment:
I'm still uncertain about naming conventions, just a note to myself that I need to make sure all of this is consistent (`authenticatorClass` vs `authenticator`)

Reply:
Should I update to authenticator for now?

Reply:
vote for authenticator

Reply:
Let's use `authenticator` for now 👍

Reply:
Got it!